dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1515,7 +1516,9 @@ private void printWesLaunchHelpBody() {
         out("");
         out("Optional parameters:");
         out("  --json <json file>                  JSON parameter file for the WES run. This may be reference an attached file");
-        out("  --attach <path, -a <path>           A list of paths to files that should be included in the WES request. (ex. -a <path1> <path2> OR -a <path1> -a <path2>)");
+        out("  --attach <path>, -a <path>          A list of paths to files that should be included in the WES request. (ex. -a <path1> <path2> OR -a <path1> -a <path2>)");
```

Comment:
Is this what was referred to by `(This would all be solved if we could send the entire workflow as a zipped bundle, that way the WES server would be able to unpack it and maintain the directory structure, but that doesn't appear to match the WES spec).` 

Referencing each file this way seems like a real pain. Maybe a new ticket could be to attach files by zip bundle but have the CLI unpack it and essentially use `--attach` under the hood on each file in it. Not sure if that makes sense, so wait for user feedback?

Reply:
Or maybe an option to point at a directory recursively instead?

Reply:
There is a TODO (https://github.com/dockstore/dockstore-cli/pull/113/files#diff-c271a67a154db53da69eacd582cb800f031f4b3f259d437fa22bdaebc0255d82L76) to allow users to specify a directory to attach and the CLI will find and attach all files to the WES request.

What I meant by: 

> (This would all be solved if we could send the entire workflow as a zipped bundle, that way the WES server would be able to unpack it and maintain the directory structure, but that doesn't appear to match the WES spec).

Is that if WES supported receiving a single zip file which contained all attachment files, and then on the WES server itself the zip files were unpacked, we wouldn't have this directory structure issue. (AGC supports sending zip files, but you must also supply a `MANIFEST.json` that points to it and the primary descriptor). But, since this isn't the current state of WES, it doesn't affect this PR.